src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/QueryResultEditorViewImpl.java

```diff
@@ -127,4 +161,50 @@ public void setErrorMessageVisible(boolean visible) {
 		errorAlert.setVisible(visible);
 	}
 
+	@Override
+	public void setProgressWidget(IsWidget progress) {
+		progressPanel.add(progress);
+	}
+
+	@Override
+	public void setEditorPanelVisible(boolean visible) {
+		this.editorPanel.setVisible(visible);
+	}
+
+	@Override
+	public void setProgressPanelVisible(boolean visible) {
+		this.progressPanel.setVisible(visible);
+	}
+	
+	@Override
+	public void showConfirmDialog(String message, final Callback callback) {
+		Bootbox.confirm(message, new ConfirmCallback() {

```

Comment:
This looks like a snippet of code that has the potential to be copy-pasted all over the app, consider refactoring to a utility.